framework/contrib/PythonFMU/build_pythonfmu

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+# This script is aimed to check and build
+# PythonFMU RAVEN compatible
+# With this script we can support all the OSs
+# we support in RAVEN
+# @author, Andrea Alfonsi
+# @date, 05/05/2021
+
+# establish conda
+../../../scripts/establish_conda_env.sh --load
+# fail if anything here fails
+set -e
+echo "start PythonFMU (RAVEN compatible) installation ......"
+# firstly lets check if the resources/binaries folders are empty or not
+# (some binaries are released with the code). At least for the current
+# operating system. Supporter are Win64, linux64, darwin64
+
+OSDIR=""
+DIRB="./pythonfmu/resources/binaries"
+if [[ "$OSTYPE" == "linux-gnu"* ]]; then
+  OSDIR="linux64"
+elif [[ "$OSTYPE" == "darwin"* ]]; then
+  OSDIR="darwin64"
+elif [[ "$OSTYPE" == "cygwin" ]]; then
+  OSDIR="linux64"
+elif [[ "$OSTYPE" == "msys" ]]; then
+  OSDIR="win64"
+elif [[ "$OSTYPE" == "win64" ]]; then
+  OSDIR="win64"
+else
+  echo "OS $OSTYPE not supported!"
+  exit 1
+fi
+
+BINARYFOLDER="$DIRB/$OSDIR"
+BUILDIT=0
+# look for empty dir
+if [ "$(ls -A $BINARYFOLDER)" ]; then
+  echo "** PythonFMU binary folder for OS '$OSDIR' has been found at $BINARYFOLDER. No build is required!"
+else
+  echo "** PythonFMU binary folder for OS '$OSDIR' has NOT been found at $BINARYFOLDER. BUILD is required!"
+  BUILDIT=1
+fi
```

Comment:
Hm, I am not sure that distributing a binary is a good idea.  (This is also a comment on everything in the framework/contrib/PythonFMU/pythonfmu/resources/binaries directory)  Also, I am not sure that the ostype check is detailed enough (Does the linux binary really work on both cygwin and all linuxes for all python versions?)
Where did these binaries come from?

Reply:
the binaries came from the PythonFMU directly (the Linux and Win) and they claim they work in cygwin too (the linux). But I think you are right. It is much safer to build them in the machine where we compile RAVEN. I removed them.